docs/completeExamples/completeExamples.md

```diff
@@ -0,0 +1,44 @@
+# Complete Examples
+A number of examples of complete events are provided below, in order to illustrate use of the schema.
+
+N.B. All entities including organisations, persons, devices and applications are imaginary.
```

Comment:
OK Done.  I've worked out how to turn word wrap on in my editor now!  #livinthedream :-D